app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/SceneService.scala

```diff
@@ -36,21 +67,59 @@ class SceneService[ProjStore: ProjectStore, HistStore](
       ForeignError.handle {
         AuthedService {
           case GET -> Root / UUIDWrapper(sceneId) / IntVar(z) / IntVar(x) / IntVar(
-                y)
-                :? BandOverrideQueryParamDecoder(bandOverride) as user =>
+                y) as user =>
             val bbox = TileUtils.getTileBounds(z, x, y)
-            val eval =
-              LayerTms.identity(
-                scenes.read(sceneId, Some(bbox), bandOverride, None))
             for {
-              fiberAuth <- authorizers.authScene(user, sceneId).start
-              fiberResp <- eval(z, x, y).start
-              _ <- fiberAuth.join.handleErrorWith { error =>
-                fiberResp.cancel *> IO.raiseError(error)
+              authFiber <- authorizers.authScene(user, sceneId).start
+              bandsFiber <- getDefaultSceneBands(sceneId)
+              footprintFiber <- getSceneFootprint(sceneId)
+              _ <- authFiber.join.handleErrorWith { error =>
+                bandsFiber.cancel *> footprintFiber.cancel *> IO.raiseError(
+                  error)
+              }
+              footprint <- footprintFiber.join
+              bands <- bandsFiber.join
+              eval = LayerTms.identity(
+                scenes.read(sceneId, Some(bbox), Some(bands), None))
+              resp <- eval(z, x, y) flatMap {
+                case Valid(tile) =>
+                  Ok(tile
```

Comment:
I don't understand the change here from `Ok(tile.renderPng.bytes, pngType` -- reverting this change fixes the new 500's being thrown by this PR and also fixes the inconsistent color preview for scenes on the map:

![image](https://user-images.githubusercontent.com/898060/53018170-a6e27400-341f-11e9-8a0a-cd8b774a83f5.png)

Reply:
Try this with a scene that doesn't start in the correct color space -- e.g. planet and MODIS go haywire if you don't rescale

Reply:
I guess since I fixed the band order I can just hit these with color correction. I'll try that instead.

Reply:
nevermind, we wrote `paintedMosaicExtentReification` for a reason